components/src/dynamo/sglang/engine_routes.py

```
@@ -0,0 +1,523 @@
+        active_ids.remove(value_id)
+
+
+def normalize_engine_route_result(result: Any) -> dict[str, Any]:
+    """Convert a configured Engine or tokenizer-manager result to a JSON object."""
+
+    if result is None:
+        return {"status": "ok"}
+    if isinstance(result, tuple) and len(result) in {2, 3}:
+        keys = ("success", "message", "num_paused_requests")
+        return {
+            key: normalize_engine_route_value(value) for key, value in zip(keys, result)
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)